kyuubi-server/src/main/resources/sql/mysql/006-KYUUBI-7028.mysql.sql

```diff
@@ -0,0 +1,16 @@
+SELECT '< KYUUBI-7028: Persist Kubernetes metadata into metastore' AS ' ';
+
+CREATE TABLE IF NOT EXISTS kubernetes_metadata(
```

Comment:
the `metadata` is a quite general concept, thus it cannot help the user understand what the table is for. I saw we use `appInfoStore` in the code, it's a good name, maybe the table name could be `k8s_app_info`, or `k8s_engine_info`, or `k8s_engine_info_store`